pkg/deviceshifu/deviceshifumqtt/deviceshifumqtt.go

```diff
@@ -171,6 +202,10 @@ func (handler DeviceCommandHandlerMQTT) commandHandleFunc() http.HandlerFunc {
 				http.Error(w, "Error to publish a message to server", http.StatusBadRequest)
 				return
 			}
+			if _, isMutexState := mutexInstructions[string(requestBody)]; isMutexState {
+				mutexBlockingMap[mqttTopic] = true
+				logger.Infof("Message %v is mutex, blocking.", requestBody)
+			}
 			logger.Infof("Info: Success To publish a message %v to MQTTServer!", requestBody)
```

Comment:
what if the topic is blocked, will this message still say successfully published?